apps/web/app/(app)/onboarding/OnboardingBulkUnsubscriber.tsx

```
@@ -0,0 +1,118 @@
+} from "@/app/api/user/stats/newsletters/route";
+import { LoadingContent } from "@/components/LoadingContent";
+
+const useNewsletterStats = () => {
+  const params: NewsletterStatsQuery = {
+    types: [],
+    filters: [],
+    orderBy: "unarchived",
+    limit: 3,
+    includeMissingUnsubscribe: true,
+  };
+  const urlParams = new URLSearchParams(params as any);
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)